utils/_context/containers.py

```diff
@@ -75,6 +82,7 @@ def __init__(
         allow_old_container=False,
         healthcheck=None,
         stdout_interface=None,
+        depends_on=[],
```

Comment:
Do not use mutable values for default args

```suggestion
        depends_on=None,
```